#!/usr/bin/env python
PACKAGE = "worldModel"

import roslib;
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("teammember_timeout", int_t, 0, "Team member time-out for updating in milliseconds", 500, 0, 100000)
gen.add("obstacle_timeout", int_t, 0, "Obstacle time-out for updating in milliseconds", 500, 0, 100000)
gen.add("ball_timeout", int_t, 0, "Ball time-out for updating in milliseconds", 1000, 0, 10000)
 
exit(gen.generate(PACKAGE, "Administration", "administration"))
